models/inscripcion.go

```diff
@@ -13,7 +13,7 @@ import (
 type Inscripcion struct {
 	Id                  int                `orm:"column(id);pk;auto"`
 	PersonaId           int                `orm:"column(persona_id)"`
-	ProgramaAcademicoId int                `orm:"column(programa_academico_id)"`
+	ProgramaAcademicoId *int               `orm:"column(programa_academico_id)"`
```

Comment:
```suggestion
	ProgramaAcademicoId int               `orm:"column(programa_academico_id);null"`
```
No entiendo por qué debe definir la variable como puntero 🤔